index.js

```diff
@@ -0,0 +1,56 @@
+// Get inputs value
+const titleInput = document.querySelector('.title-value');
+const authorInput = document.querySelector('.author-value');
+
+let booksObject = [];
+
+// Pouplate the HTML with the books
+const populateHTML = (books) => {
+  const booksContainer = document.createElement('div');
+  booksContainer.innerHTML = '';
+  books.forEach((book) => {
+    booksContainer.innerHTML += `
+            <div class="book" id="book${book.id}">
+                <p class="title">${book.title}</p>
+                <p class="author">${book.author}</p>
+                <button type="button" class="remove-btn" id="${book.id}">Remove</button>
+                <hr/>
+            </div>
+            `;
+  });
+  document.querySelector('.books').appendChild(booksContainer);
+};
+
+populateHTML(booksObject);
```

Comment:
- [ ] when I refresh the page, the list is empty this is because at this point `booksObject` is an empty array, you can fix this by either making your `booksObject` variable the value of local storage or empty array if localstorage is empty i.e `const booksObject = JSON.parse(localStorage.getItem('books')) || [];` there are many other ways you can try your hands :+1: